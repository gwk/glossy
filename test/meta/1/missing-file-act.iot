{
  'cmd': '$PROJ/iotest/__main__.py -no-times test-meta/1/$NAME.iot',
  'code': 1,
  'out_mode': 'match',
}
