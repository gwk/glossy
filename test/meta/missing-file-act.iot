{
  'cmd': '$PROJ/iotest.py -no-times test-meta/$NAME.iot',
  'code': 1,
  'links' : {'test-meta'},
  'out_mode': 'match',
}
