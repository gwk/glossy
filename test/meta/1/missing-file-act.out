| test-meta/1/missing-file-act
|
~ iotest: test did not output expected file: .+/iotest/_build/test-meta/1/missing-file-act/nonexistant.txt
| ================================================================
|
|
| ################################################################
| TESTS FOUND: 1; SKIPPED: 0; FAILED: 1.
